x-pack/plugins/observability/public/pages/alerts/alerts_flyout/index.tsx

```diff
@@ -97,14 +94,8 @@ export function AlertsFlyout({
       description: alertData.active ? 'Active' : 'Recovered',
     },
     {
-      title: i18n.translate('xpack.observability.alertsFlyout.severityLabel', {
-        defaultMessage: 'Severity',
-      }),
-      description: <SeverityBadge severityLevel={alertData.fields[ALERT_SEVERITY_LEVEL]} />,
-    },
-    {
-      title: i18n.translate('xpack.observability.alertsFlyout.triggeredLabel', {
-        defaultMessage: 'Triggered',
+      title: i18n.translate('xpack.observability.alertsFlyout.lastUpdatedLabel', {
```

Comment:
@weltenwort I spotted this small bug and fixed as part of this PR.

Reply:
good catch, thank you